src/services/web3/protection/positions.ts

```diff
@@ -332,11 +332,11 @@ export const fetchProtectedPositions = async (
     rawPositions
   );
 
-  const positionsAPR = await fetchPoolAprs(
-    pools,
-    rawPositions,
-    liquidityProtectionContract
-  );
+  // const positionsAPR = await fetchPoolAprs(
```

Comment:
all this commented out code feels unintentional

Reply:
It is intentional since we are not displaying aprs by week and day so no need to fetch them

Reply:
but why comment out code and not remove it? We have version control - we can find the code if we need it.